Polish/przygoda/jedzenie/korwin/korwin.md

```diff
@@ -4,4 +4,6 @@ Wygrałeś
 Niedługo koronacja Korwina na krula wszechświata.
 Jaki prezent podarujesz swojemu monarsze?
 
-[Złoty rewolwer](prezent/rewolwer.md)
\ No newline at end of file
+[Złoty rewolwer](prezent/rewolwer.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.